target/linux/ath79/dts/ar9341_openmesh_om2p-hs-v1.dts

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9341_openmesh_om2p-hs.dtsi"
+
+/ {
+	compatible = "openmesh,om2p-hs", "qca,ar9341";
```

Comment:
-v1 missing here.

Reply:
Good catch. This was introduced when I've changed the v1 devices to have the variant "v1" included - which was (unfortunately) after I returned the borrowed OM2P-HS v1. So this would definitely would have caused problems in the field.

Was now fixed